frontend/src/profile-registration/EmailInput.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export class EmailInput extends Component {
+    render() {
+        const { onChange } = this.props
+        const id = uuidv4()
```

Comment:
using unique ID library is odd. You can name id as a combination of component name and element name